test/gtest/uct/test_md.cc

```diff
@@ -436,6 +443,9 @@ UCS_TEST_SKIP_COND_P(test_md, alloc,
             if (mem_type == UCS_MEMORY_TYPE_HOST) {
                 memset(address, 0xBB, size);
             }
+
+            uct_md_mkey_pack_v2(md(), mem.memh, &pack_params,&key);
```

Comment:
1. rename params to alloc_params
2. space before &key
3. do we need to pass the buffer to pack to?

Reply:
I don't think I understand (3), if we don't pass `key` (the key buffer) mkey_pack_v2 will return INVALID_PARAM status
I will add an assertion for the returned status

Reply:
pls ignore (3), i misread the code